pom.xml

```diff
@@ -16,6 +16,10 @@
   <packaging>maven-plugin</packaging>
 
 
+  <prerequisites>
+    <maven>3.6.3</maven>
```

Comment:
How will this affect those who have an older version. Will they get a warning or will the build fail?
Is that version actually required?

Reply:
The build will fail and tell them to use 3.6.3 or better.

Technically, 3.8.x is now also end of life, but all the core plugins are only enforcing 3.6.3 or better.  I think that errors on the side of caution in general as 3.9.x generally required work to update to, and I presume many corporations like mine still allow 3.8 versions.  By the time maven was otherwise at 3.6.x things had changed enough that most older builds would fail anyways but not be all that clear as to why.

To test that out yourself just toss any high number there the try to execute the build to see the specific output.